### PR TITLE
Template legacy hookname duplicated

### PR DESCRIPTION
**Pull request by bordoni**

### CODE REVIEW

File: src/Tribe/Template.php [hunk 1]
```diff
@@ -872,22 +872,30 @@ public function template( $name, $context = [], $echo = true ) {
...
+			 *
+			 * @since        4.7.20
+			 *
+			 * @deprecated   4.11.0
+			 *
+			 * @param string $file     Complete path to include the PHP File
+			 * @param array  $name     Template name
+			 * @param self   $template Current instance of the Tribe__Template
+			 */
+			do_action_deprecated(
```
Comment (lucatume) [inline src/Tribe/Template.php:892]: Do we have an idea about how old is *the most recent code* using the legacy hook names?
If that code is not, at least, 1.5yrs old, then I would add a todo task and not throw a deprecation notice for now.

File: src/Tribe/Template.php [hunk 2]
```diff
@@ -919,22 +927,30 @@ public function template( $name, $context = [], $echo = true ) {
...
+			 *
+			 * @since      4.7.20
+			 *
+			 * @deprecated 4.11.0
+			 *
+			 * @param string $file     Complete path to include the PHP File
+			 * @param array  $name     Template name
+			 * @param self   $template Current instance of the Tribe__Template
+			 */
+			do_action_deprecated(
```
Comment (lucatume) [inline src/Tribe/Template.php:947]: Same note and question about code "age" as above hee.

File: src/Tribe/Template.php [hunk 3]
```diff
@@ -968,23 +984,31 @@ public function template( $name, $context = [], $echo = true ) {
...
+			 * @since        4.7.20
+			 *
+			 * @deprecated   4.11.0
+			 *
+			 * @param string $html     The final HTML
+			 * @param string $file     Complete path to include the PHP File
+			 * @param array  $name     Template name
+			 * @param self   $template Current instance of the Tribe__Template
+			 */
+			$html = apply_filters_deprecated(
```
Comment (lucatume) [inline src/Tribe/Template.php:1005]: Same note and question about code and age here.

File: src/Tribe/Template.php (hunk 1 above)
Comment (Camwyn) [reply src/Tribe/Template.php:895]: The hooks were added in TEC 5.0.0, so not 1.5 years old?

Comment (lucatume) [reply src/Tribe/Template.php:895]: So I would not deprecate them yet.

**Pull request merged**